OSDOCS-17161-2:(CQA) APP-2: Deployments - Attachment3 (#117477)
openshift/openshift-docs: modules/deployments-rolling-back.adoc

[id="deployments-rolling-back_{context}"]
= Rolling back a deployment

> [role="_abstract"]
> Rollbacks revert an application back to a previous revision and can be performed using the REST API, the CLI, or the web console.

Reviewer: Would this be stronger in the active voice?

```suggestion
[role="_abstract"]
To revert an application to a previous revision, you can perform a roll back  by using the REST API, the CLI, or the web console.
```
